app/components/ContextMenu/index.tsx

```diff
@@ -146,7 +146,7 @@ const ContextMenu: React.FC<Props> = ({
           );
         }}
       </Menu>
-      {(rest.visible || rest.animating) && (
+      {(rest.visible || rest.animating) && rest["aria-label"] !== t("Submenu") && (
```

Comment:
This is to prevent the issue of _double backdrop_, one from parent and other from sub-menu.